contrib/whois/template.py

```diff
@@ -0,0 +1,38 @@
+pkgname = "whois"
+pkgver = "5.5.18"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_build_args = [
+    "CONFIG_FILE=/etc/whois.conf",
+    "HAVE_ICONV=1",
+]
+make_install_target = "install-whois"
+make_install_args = ["install-mkpasswd"]
+hostmakedepends = [
+    "gettext",
+    "gmake",
+    "perl",
+    "pkgconf",
+]
+makedepends = [
+    "libidn2-devel",
+]
+depends = ["!expect"]
```

Comment:
don't need this (even if i didn't resolve the conflict, you get an implicit conflict through two versioned providers of the `cmd:`)